### PR TITLE
[EWL-4866] SG2 | Broken Markup for Tools Block on Topic Page

### PR DESCRIPTION
**Pull request by JaeBea**
## Ticket(s)

**Jira Ticket**
- [EWL-4866 SG2 | Broken Markup for Tools Block on Topic Page](https://issues.ama-assn.org/browse/EWL-4866)

## Description

The tools block on any topic page should render with a h5 heading tag and class "ama__h5". Currently on `develop` the tools block heading and learn more heading shows broken HTML. The heading renders as `<h>` (https://americanmedicalassociation.github.io/ama-style-guide-2/?p=pages-topic).

This update corrects the broken markup by defining "level" and "class" in the topic page json for heading atoms related to the tools block. Backstop.json has been updated to reference the correct wrapper dive for tools block. This will ensure any regression is found during tests for this block.

## To Test
- [ ] Pull `develop` branch.
- [ ] Run `gulp serve` to load style guide.
- [ ] Go to http://localhost:3000/?p=pages-topic.
- [ ] Inspect heading element for Tools block to see broken heading markup.
- [ ] Pull `feature/EWL-4866-broken-markup-tools-block` branch.
- [ ] Run `gulp serve` to load style guide.
- [ ] Go to http://localhost:3000/?p=pages-topic.
- [ ] Observe updated tools block headings. You may inspect element to see fixed HTML.

## Visual Regressions

Only 6 fails occur when running `backstop test`. These failures are from a difference in exact date specified on "membership" molecules and the tool block reference images . Those new reference images have been committed with this PR.

## Relevant Screenshots/GIFs

**Broken HTML Headers**
screenshot-topicpage-toolsheaders
screenshot-topicpage-toolsheaders-inspector

**Fixed HTML Headers**
screenshot-topicpage-toolsheaders-fixed

## Remaining Tasks

N/A

## Additional Notes

- Ticket needs to be created to address missing backstop reference images for other molecules/organisms that show "not found" during test.

---

### CODE REVIEW

**Comment by labbydev** [comment]
@JaeBea how is this related to PR #165 they both seem to be doing the same thing? (or at least similar things).

**Comment by JaeBea** [comment]
@labbydev The PR you noted updated the tools organism to have h5 headings in SG2 and d8.

This update adds missing attributes to the json for the topics page tool organism as only part of the attributes for the header atom were defined. The partially defined attributes seemed to have caused the markup to be broken on SG2 only. I have added another screenshot to clarify what was going on in the html.

**Comment by mrspete** [comment]
@JaeBea please resolve the conflicts and resubmit.  Thanks!

**Comment by JaeBea** [comment]
@mrspete and @labbydev merge conflict seems to be resolved.

File: styleguide/source/assets/scss/01-atoms/_unordered-list.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+ul {
+  li {
+    @include gutter($margin-left-full...);
```
Comment (labbydev) [inline styleguide/source/assets/scss/01-atoms/_unordered-list.scss:3]: This specifically is causing a number of issues in D8 and making things like the admin menu and the IPE look broken. 

screen shot 2018-04-24 at 11 07 25 am

**Review by labbydev — CHANGES_REQUESTED** [review]
Please _only include_ the code that is relevant to this specific ticket. 

`README`, `standards`, `package.json`, `date-block.json`, `date-block.twig`, `article-stub.twig`, `promo.twig`, `lead-news.json`, `lead-news.twig`, `masthead.json`, `masthead.twig`, `news.json`,  `press-release.json`, `style-guide-welcome.json`, `style-guide-welcome.twig`, `topic.json`, `wayfinder.js`, `_unordered-list.scss`, `_lead-news.scss`, `_promo.scss`, `_ribbon-dropdown.scss`, `_tool.scss`, `_ribbon.scss` updates that are not specific to this ticket should _not_ be included in this PR.

**Comment by JaeBea** [comment]
After discussion with team this PR will be closed.

**Pull request closed without merging**